lib/matplotlib/collections.py

```diff
@@ -1396,29 +1450,22 @@ def __init__(self, segments,     # Can be None.
            "interior" can be specified by appropriate usage of
            `~.path.Path.CLOSEPOLY`.
         **kwargs
-            Forwareded to `.Collection`.
+            Forwarded to `.Collection`.
         """
-        if colors is None:
-            colors = mpl.rcParams['lines.color']
-        if linewidths is None:
-            linewidths = (mpl.rcParams['lines.linewidth'],)
-        if antialiaseds is None:
-            antialiaseds = (mpl.rcParams['lines.antialiased'],)
-
-        colors = mcolors.to_rgba_array(colors)
+        argnames = ["linewidths", "colors", "antialiaseds", "linestyles",
+                    "offsets", "transOffset", "norm", "cmap", "pickradius",
+                    "zorder", "facecolors"]
+        if args:
+            argkw = {name: val for name, val in zip(argnames, args)}
+            kwargs.update(argkw)
+            cbook.warn_deprecated(
+                "3.4", message="In a future release, all LineCollection "
+                "arguments other than the first, 'segments', will be "
+                "keyword-only arguments."
```

Comment:
This needs to mention when it was deprecated and when it's going to be removed.
```suggestion
                "3.4", message="Since %(since)s, passing LineCollection "
                "arguments other than the first, 'segments', as positional arguments "
                "is deprecated, and they will become keyword-only arguments %(removal)s."
```